R/workflow.R

```diff
@@ -69,54 +69,56 @@ print.workflow <- function(x, ...) {
 
 print_header <- function(x) {
   if (x$run) {
-    fit <- " [fit]"
+    fit <- " [trained]"
   } else {
     fit <- ""
   }
 
-  cat_line(glue::glue("<workflow{fit}>"))
+  cat(cli::rule(glue::glue("model workflow {fit}")))
```

Comment:
Is the rule really that useful as the first line? When we talked before, I thought it was going to be something like:

```r
---- Workflow ---------
- Preprocessor: Recipe
- Model: Linear regression

---- Recipe ---------
- step1
- step2

---- Model ---------
print model spec
```

Reply:
Double rule for "Workflow"